aldryn_newsblog/cms_wizards.py

```diff
@@ -93,38 +93,25 @@ def __init__(self, **kwargs):
 
     def save(self, commit=True):
         article = super(CreateNewsBlogArticleForm, self).save(commit=False)
-
-        # Set owner to current user
         article.owner = self.user
 
-        # If 'content' field has value, create a TextPlugin with same and add
```

Comment:
the reason for content being set before reversion is because otherwise revision wouldn't include the content.
I think is ok now because plugins (content) are not really supported in revision since the cms dropped support for it.

Reply:
Ahh gotcha. In fact that was a bit obscure to me. Thanks for the explanation!